src/rust/plugin-registry/src/nomad/client.rs

```diff
@@ -137,6 +139,24 @@ impl NomadClient {
         .await
         .map_err(NomadClientError::from)
     }
+
+    #[tracing::instrument(skip(self, job_name, namespace), err)]
+    pub async fn delete_job(
```

Comment:
congrats you're now the second person to touch the nomad-rs stuff. it's a lil wonky but not too awful